recipes/oommf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TCLTKVERSION=8.5
+CONDA_ROOT_PATH=$(conda info --root)
+
+# Compile OOMMF
+export OOMMF_TCL_CONFIG=${PREFIX}/lib/tclConfig.sh
+export OOMMF_TK_CONFIG=${PREFIX}/lib/tkConfig.sh
+cd oommf
+tclsh${TCLTKVERSION} oommf.tcl pimake distclean
```

Comment:
You mean that we use the default package from conda? Does conda-forge have a better one? If so, how do we choose to use that? (Sorry for naive questions, very new to conda-forge.)

[The tclsh in the path in the example above is provided by OSX, I think.]